forge-gui/res/tokenscripts/g_1_1_insect_infect.txt

```diff
@@ -0,0 +1,7 @@
+Name:Insect Token
```

Comment:
why are we adding tokenscripts for nothing?

Reply:
@tool4ever see other MRs about the old non Phyrexian token

Reply:
I didn't wanted to delete this one :/

I thought it was another g_1_1_insect

Reply:
I'm presonally not a fan of changing all of these tokens away from what we had them. So i don't think i really have further input.

Reply:
Tokens aren't cards and therefore aren't errata'd so it still officially exists as a 1/1 green Insect token, even if it isn't used at the moment.